lib/babel-config.js

```diff
@@ -20,6 +21,8 @@ module.exports = function (env) {
     reactRequire
   ]
 
+  if (env === 'instrumented') { plugins.push(istanbul) }
```

Comment:
Hmmm. Should this be it's own argument? Just `--instrument`.
 `env` is primarily used for test/dev/staging/production